templates/magazine/_federated_info.html.twig

```diff
@@ -1,6 +1,25 @@
-{% if magazine.apId %}
-    <div class="alert alert__info">
-        <p>{{ 'federated_magazine_info'|trans }} <a
-                    href="{{ magazine.apProfileId }}">{{ 'go_to_original_instance'|trans }}</a></p>
-    </div>
-{% endif %}
\ No newline at end of file
+{% if magazine.apId %}   {# I.e. if we're federated #}
+    {% if entries is defined and entries and not entries.hasNextPage %}
+        {# Then show a link to original if we're at the end of content #}
+        <div class="alert alert__info">
+            <p>
+                {{ 'federated_magazine_info'|trans }} <a href="{{ magazine.apProfileId }}">{{ 'go_to_original_instance'|trans }}</a>
+            </p>
+        </div>
+    {% endif %}
+
+    {% if not magazine_has_local_subscribers(magazine) %}
+        {# Also show a warning if we're not actively receiving updates #}
+        {% set lastOriginUpdate = magazine.lastOriginUpdate %}
+        <div class="alert alert__info">
+            {% if lastOriginUpdate is not null %}
+                {% set currentTime = "now"|date('U') %}
+                {% set secondsDifference = currentTime - (lastOriginUpdate|date('U')) %}
+                {% set daysDifference = (secondsDifference / 86400)|round(0, 'floor') %}
+                <p>{{ 'disconnected_magazine_info'|trans({'%days%': daysDifference}) }}</p>
+            {% else %}
+                <p>{{ 'always_disconnected_magazine_info'|trans }}</p>
+            {% endif %}
+        </div>
+    {% endif %}
+{% endif %}
```

Comment:
> Subscribe to start receiving updates.

Doesn't really make sense if the user isn't logged in. Maybe split the text in two so the second sentence is only displayed if the user is logged in.

Reply:
Okay done